Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mjstealey/hs_docker_base:1.7.2
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
-
+RUN sudo pip install pycrs
```

Comment:
@pkdash It would be good to set this to a specific version. `pycrs==XX.YY.ZZ`

If what's installed by the default call is adequate, or if it does not matter, I'll use the version I find when building this for migration to the **hs_docker_base** image.